modules/misunderstood/src/views/full/MainScreen/ChatPreview.tsx

```diff
@@ -1,28 +1,31 @@
-import classnames from 'classnames'
-import React from 'react'
+import classnames from "classnames";
+import React from "react";
+import pick from "lodash/pick";
 
-import { ContextMessage } from '../../../types'
+import { ContextMessage } from "../../../types";
 
-import style from './style.scss'
+import style from "./style.scss";
```

Comment:
Please make sure you have Prettier and TSLint installed on VSCode, it should be single quotes, and no ;

Reply:
Fixed these